android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -228,10 +228,14 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
             default:
         }
 
-        if(session.isBookmarked()) {
+        if(session.isBookmarked() && (Color.parseColor(session.getTrack().getFontColor()) == Color.WHITE)) {
```

Comment:
But why including a black and white icon? You can tint it. And no need to check if color is white or black. Just tint the icon with font color